tools/introspection/CMakeLists.txt

```diff
@@ -55,10 +57,19 @@ target_link_libraries(iceoryx_introspection
     ncurses
 )
 
+
+
+# Find tinfo library for APPLE, for other platforms its part of package Curses
+if(CMAKE_SYSTEM_NAME MATCHES Darwin)
```

Comment:
we have a platform detection in iceoryx_utils, just `include(IceoryxPlatformDetection)` and then you can do `if(APPLE)` ... maybe a ``find_package(iceoryx_utils REQUIRED)` is necessary before using the include

you could also add all OSes here and make an empty LIB_TINFO for QNX,  then the if statement below could be removed and target_link_libraries could be called directly

Reply:
This makes cmake file a little simpler.
Taken care